tasks/build.js

```diff
@@ -24,7 +24,11 @@ module.exports = function(grunt) {
         // If the serve command was passed,
         // don't let the task end, because
         // the serve will end...
-        if (grunt.option('serve')) {
+        if (grunt.option('serve') && !grunt.option('watcher')) {
+            buildTasks.push('build:keepAlive');
+        }
+
+        if (grunt.option('serve') && grunt.option('watcher')) {
```

Comment:
If we require "serve" option for the watcher to work it should be included in the readme, otherwise I would remove it from this "if".